mlx_lm/generate.py

```diff
@@ -322,14 +323,17 @@ def generate_step(
         kv_group_size (int): Group size for KV cache quantization. Default: ``64``.
         quantized_kv_start (int): Step to begin using a quantized KV cache.
            when ``kv_bits`` is non-None. Default: ``0``.
-        prompt_prorgress_callback (Callable[int, int]): A call-back which takes the
+        prompt_progress_callback (Callable[int, int]): A call-back which takes the
            prompt tokens processed so far and the total number of prompt tokens.
+       input_embeddings (mx.array, optional): Input embeddings to use in place of
```

Comment:
(indentation is off)
```suggestion
        input_embeddings (mx.array, optional): Input embeddings to use in place of
```

Reply:
Oops! Good catch. I guess black/pre-commit doesn't do docstrings